shared/uavobjectdefinition/hwplayuavosd.xml

```diff
@@ -15,7 +15,7 @@
 		<field name="USB_HIDPort" units="function" type="enum" elements="1" options="USBTelemetry,RCTransmitter,Disabled" defaultvalue="USBTelemetry">
 			<description/>
 		</field>
-		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" defaultvalue="Disabled">
+		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" parent="HwShared.USB_VCPPort" defaultvalue="Disabled">
```

Comment:
I almost included UAVO-to-MSP on this board. At that point it could act as an MSP relay, but I couldn't really see a use-case for it.